### ISSUE TITLE
Some Acceptance Tests Fail Locally

### ISSUE THREAD

**Issue by thavelick** [thoughtbot/shoulda-matchers#671]
I've seen several issues here I think I can tackle, so I forked the project and ran through the instructions in `CONTRIBUTING.md`.  Unfortunately, for me, the acceptance tests fail.  It appears to be something wrong in trying to setup a rails test environment inside the tests.  These are the tests that fail:

```
  Failed examples:

  rspec ./spec/acceptance/rails_integration_spec.rb:81 # shoulda-matchers integrates with Rails in a project that combines both RSpec and Test::Unit
  rspec ./spec/acceptance/rails_integration_spec.rb:61 # shoulda-matchers integrates with Rails in a project that uses Spring
  rspec ./spec/acceptance/rails_integration_spec.rb:37 # shoulda-matchers integrates with Rails in a project that uses the default test framework
  rspec ./spec/acceptance/rails_integration_spec.rb:49 # shoulda-matchers integrates with Rails in a project that uses RSpec
```

Here is a gist of the full test run: https://gist.github.com/thavelick/383539149fec74d94eec

The mystery for me is why travis-ci is happy when this happens locally.  I suspect I'm doing something dumb, but hopefully someone can point me in the right direction, and we can update `CONTRIBUTING.md` to help out future developers.

**Comment by mcmire** [comment]
Woah, that's kinda crazy. You definitely shouldn't be getting a segfault when attempting to run the acceptance tests. These warnings in the output are also interesting:

```
Ignoring binding_of_caller-0.7.2 because its extensions are not built.  Try: gem pristine binding_of_caller --version 0.7.2
       Ignoring capybara-webkit-1.3.0 because its extensions are not built.  Try: gem pristine capybara-webkit --version 1.3.0
       Ignoring debug_inspector-0.0.2 because its extensions are not built.  Try: gem pristine debug_inspector --version 0.0.2
...
```

As far as I can tell, you're running the tests correctly. Maybe you need to recompile Ruby...?

**Comment by thavelick** [comment]
I'll give that a shot, and try from a different machine. Thanks for taking a look.

**Comment by thavelick** [comment]
Finally got back to this, and I got the tests all passing on a fresh VM!  I'm drawing up some additional documentation that should help new people get up and running a little faster, and I'll submit a pull request for it here shortly.

**Comment by mcmire** [comment]
Sweet, glad to hear. Thanks @thavelick.

**Comment by mcmire** [comment]
@thavelick Ping... what were the things you needed to do in order to run the tests?

**Comment by thavelick** [comment]
@mcmire, 

Sorry for the delays.  This PR has instructions for what I had to do: https://github.com/thoughtbot/shoulda-matchers/pull/689

I didn't add any mac instructions, as I'm afraid my mac is in a weird state, and it's not quite as easy to spin up a mac vm.

Let me know your thoughts.

**Comment by mcmire** [comment]
Closed by #689.
